src/psij/web/lib.js

```diff
@@ -415,13 +415,22 @@ $(document).ready(function() {
     }, 500);
 });
 
+var CUSTOMIZATION = { projectName: "PSI/J"};
+
```

Comment:
Was this intentional?

Reply:
not quite.  i just needed it for testing.

Could I do this?
var CUSTOMIZATION = CUSTOMIZATION || { projectName: "PSI/J"};

The benefit of this would be that the code base would work out of the box without the user having to add a customization.js

Also, it wouldn't overwrite any CUSTOMIZATION that someone has already defined.  sound okay?

Reply:
If you are worried about git overwriting local changes, git is really rather careful not to do that. It will only do so only if you force it to.

To your second point, if it's testing code, it probably shouldn't be in git, unless it's easily reusable by other developers. Even so, it should not be in lib.js which gets served to every viewer of the dashboard, but likely in some separate file.

Reply:
Ok, I removed the testing code and the customization line as you suggested.